### ISSUE TITLE
Add README

### ISSUE THREAD

**Issue by matthewjasper** [rust-lang-nursery/reference#2]
See title.

Some of the questions that I would hope it would answer are:
* All of the usual things about building and contributing.
* If and how are larger contributions and rewrites accepted?
* What are the main goals for improving the reference?

**Comment by steveklabnik** [comment]
Absolutely!

**Comment by est31** [comment]
Bump. Just explained someone on IRC how to invoke mdbook to build the reference, which wouldn't have been required had there been a README.

**Comment by brauliobezerra** [comment]
I didn't put the `--git https://github.com/azerupi/mdBook.git` when installing mdbook. Is it required?

**Comment by steveklabnik** [comment]
@brauliobezerra no, it's not; any version of gitbook works, and there's no real reason to prefer the version from git.
